CMakeLists.txt

```diff
@@ -15,7 +15,12 @@ set(PARAKEET_HEADER
 	${PARAKEET_HEADER_ROOT}/ScanDataXY.h
 	${PARAKEET_HEADER_ROOT}/SerialPort.h
 	${PARAKEET_HEADER_ROOT}/util.h
+	${PARAKEET_HEADER_ROOT}/internal/SensorResponse.h
+	${PARAKEET_HEADER_ROOT}/internal/SensorResponseParser.h
 	${PARAKEET_HEADER_ROOT}/internal/SerialPortHelper.h
+	${PARAKEET_HEADER_ROOT}/internal/exceptions/NotConnectedToSensorException.h
```

Comment:
Since we expect a user to be able to catch the exception, this wouldn't be internal. Things on the internal namespace would be classes/objects a user of our code ever needs to interact with (in our view). 

- [ ] Move exceptions out of the internal namespace

Reply:
Not sure why I had put them there. Good catch.